AnnoyancesFilter/sections/popups.txt

```diff
@@ -138,6 +138,7 @@ mercola.com##.modal-wrapper
 renault.ru##.k-popinInfo_background
 swyrl.tv##.tsch--popup-wrapper .tsch--newsletter-panel
 mengen.gen.tr###whatsappIcon
+hype.tech#?#body > div:has( > div input[placeholder="E-mail"])
```

Comment:
`hype.tech#?#body > div:not([id]):not([class]) input[placeholder="E-mail"]:upward(5)`